tower-grpc-build/Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = "MIT"
 
 [dependencies]
 codegen = { git = "https://github.com/carllerche/codegen" }
-prost-build = "0.4"
+prost-build = "0.5"
```

Comment:
This is the offending change. When I put it back to `"0.4"` everything else builds fine.
That's as far as I was able to get in the short amount of time I had. Might be able to take another look later.